src/Constants.hs

```diff
@@ -1,7 +1,10 @@
-module Constants (size, initialScore) where
+module Constants (size, initialScore, twoPercentChance) where
 
 size :: Int
 size = 4
 
 initialScore :: Int
 initialScore = 0
+
+twoPercentChance :: Int
+twoPercentChance = 8 -- 8/10 = 80% chance of getting a 2
```

Comment:
Where do you get 80% percent for getting 2? Please add your research for this percent